tests/test_estimators/test_gmm_naive_bayes.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+from sklearn.utils import estimator_checks
+
+from sklego.common import flatten
+from sklego.naive_bayes import GaussianMixtureNB
+from sklego.testing import check_shape_remains_same_classifier
+from tests.conftest import nonmeta_checks, general_checks
+
+
+@pytest.mark.parametrize("test_fn", flatten([
+    nonmeta_checks,
+    general_checks,
+    estimator_checks.check_classifier_data_not_an_array,
```

Comment:
I guess you're not using all `estimator_checks`? It might be good to add a comment about which ones not and why.

Reply:
I've put in all `classifier_checks` and ... it passes.